pkg/pool-weighted/test/foundry/LBPMigrationRouter.t.sol

```diff
@@ -0,0 +1,409 @@
+        IERC20(pool).approve(address(lbpMigrationRouter), lbpBPTBalanceBefore);
+
+        (IERC20[] memory lbpTokens, TokenInfo[] memory lbpTokenInfo, , uint256[] memory lbpBalancesBefore) = vault
+            .getPoolTokenInfo(pool);
+
+        uint256[] memory weights = [50e16, uint256(50e16)].toMemoryArray();
+        uint256[] memory initBalances = [poolInitAmount / 2, poolInitAmount / 3].toMemoryArray();
+        PoolRoleAccounts memory poolRoleAccounts = PoolRoleAccounts({
+            pauseManager: makeAddr("pauseManager"),
+            swapFeeManager: makeAddr("swapFeeManager"),
+            poolCreator: address(0)
+        });
+
+        (IWeightedPool weightedPool, ) = lbpMigrationRouter.migrateLiquidity(
+            ILBPool(pool),
+            initBalances,
+            0, // minAddBptAmountOut
+            new uint256[](2), // minRemoveAmountsOut
+            excessReceiver,
+            ILBPMigrationRouter.WeightedPoolParams({
+                name: POOL_NAME,
+                symbol: POOL_SYMBOL,
+                normalizedWeights: weights,
+                roleAccounts: poolRoleAccounts,
+                swapFeePercentage: DEFAULT_SWAP_FEE_PERCENTAGE,
+                poolHooksContract: address(0),
+                enableDonation: true,
+                disableUnbalancedLiquidity: true,
+                salt: bytes32(0)
+            })
+        );
+
+        vm.stopPrank();
+
+        (IERC20[] memory tokens, TokenInfo[] memory tokenInfo, , ) = vault.getPoolTokenInfo(address(weightedPool));
+
+        // Check pool creation parameters
+        {
+            assertEq(IERC20Metadata(address(weightedPool)).name(), POOL_NAME, "Incorrect pool name");
+            assertEq(IERC20Metadata(address(weightedPool)).symbol(), POOL_SYMBOL, "Incorrect pool symbol");
+
+            assertEq(tokens.length, lbpTokens.length, "Token arrays length mismatch");
+            assertEq(tokenInfo.length, lbpTokenInfo.length, "Token info arrays length mismatch");
+            for (uint256 i = 0; i < tokenInfo.length; i++) {
+                assertEq(address(tokens[i]), address(lbpTokens[i]), "Token address mismatch");
+
+                assertEq(uint256(tokenInfo[i].tokenType), uint256(lbpTokenInfo[i].tokenType), "Token type mismatch");
+                assertEq(
+                    address(tokenInfo[i].rateProvider),
+                    address(lbpTokenInfo[i].rateProvider),
+                    "Rate provider address mismatch"
+                );
+                assertEq(tokenInfo[i].paysYieldFees, lbpTokenInfo[i].paysYieldFees, "Pays yield fees mismatch");
+            }
+
+            PoolConfig memory poolConfig = vault.getPoolConfig(address(weightedPool));
+            assertEq(poolConfig.staticSwapFeePercentage, DEFAULT_SWAP_FEE_PERCENTAGE, "Incorrect swap fee percentage");
+            assertEq(
+                poolConfig.liquidityManagement.disableUnbalancedLiquidity,
+                true,
```

Comment:
This one should actually be false. It's just a regular weighted pool now, so there's no reason to disable unbalanced liquidity.